src/Uno.UI/Generated/3.0.0.0/Microsoft.UI.Xaml.Automation.Peers/AutomationPeer.cs

```diff
@@ -14,14 +14,14 @@ public partial class AutomationPeer : global::Microsoft.UI.Xaml.DependencyObject
 		// Forced skipping of method Microsoft.UI.Xaml.Automation.Peers.AutomationPeer.EventsSource.get
 		// Forced skipping of method Microsoft.UI.Xaml.Automation.Peers.AutomationPeer.EventsSource.set
 		// Skipping already declared method Microsoft.UI.Xaml.Automation.Peers.AutomationPeer.GetPattern(Microsoft.UI.Xaml.Automation.Peers.PatternInterface)
-#if __ANDROID__ || __IOS__ || IS_UNIT_TESTS || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
+#if false
```

Comment:
Was this change part of this PR? To me this seems like a different change

Reply:
I noticed the error was showing up when the control was used, which was quite confusing for the user. So instead, I adjusted this to be just a warning for now (until we properly implement automation, which should be soon)